docs/install/operator/configuring-eventing-cr.md

```diff
@@ -243,3 +244,29 @@ metadata:
 spec:
   defaultBrokerClass: MTChannelBasedBroker
 ```
+
+## System Resource Settings
+
+The operator custom resource allows you allows you to configure system resources for the Knative system containers.
```

Comment:
Where does someone find or edit this CR? Is `operator` the literal name of the CR or is this referring to KnativeEventing?